src/protocolsupport/protocol/packet/middleimpl/serverbound/play/v_pe/Interact.java

```diff
@@ -35,7 +38,7 @@ public void readFromClientData(ByteBuf clientdata) {
 		switch (peAction) {
 			case INTERACT: {
 				if ((target != null) && (target.getType() != NetworkEntityType.ARMOR_STAND)) {
```

Comment:
Yeah good point. If it doesn't, we also might just ditch the old interact without position packet. might wamt to check if it is actually still in use.